pulsar-client-cpp/lib/Consumer.cc

```diff
@@ -23,6 +23,17 @@ namespace pulsar {
 
 const std::string EMPTY_STRING;
 
+BrokerSideConsumerStats::BrokerSideConsumerStats():validTill_(now()) {};
```

Comment:
we do that in ClientConnection

                            boost::posix_time::ptime currentTime = now() + milliseconds(consumerStatsTTLMs_);
                            consumerStatsPromise.setValue(BrokerSideConsumerStats(consumerStats, currentTime));